### PR TITLE
Instructions for installing on CentOS + minor edits

### PR DESCRIPTION
**Pull request by linuxfreeordie**
I added instructions alongside the Ubuntu instructions for Centos users, and tested it from a fresh centos 7 server. I also added a couple other notes on things that tripped me up, postgres not being initialized, and the systemd files not having the right PATH and GEM_PATH set.

### CODE REVIEW

File: Running-Mastodon/Production-guide.md [hunk 1]
```diff
@@ -98,22 +98,53 @@ It is recommended to create a special user for mastodon on the server (you could
...
+
+    sudo yum install redis rubygem-redis
+
 ## Postgres
 
+### Ubuntu / Debian
+
     sudo apt-get install postgresql postgresql-contrib
 
-Setup a user and database for Mastodon:
```
Comment (yiskah) [inline Running-Mastodon/Production-guide.md]: Why did you remove this line from the Ubuntu section and keep it in the CentOS section?

File: Running-Mastodon/Production-guide.md [hunk 2]
```diff
@@ -98,22 +98,53 @@ It is recommended to create a special user for mastodon on the server (you could
...
+
+    sudo yum install postgresql-server postgresql postgresql-contrib postgresql-devel
+
+Setup postgres and add user and database for Mastodon:
+
+    sudo postgresql-setup initdb
+    sudo systemctl start postgresql
+    sudo systemctl enable postgresql
 
     sudo su - postgres
```
Comment (yiskah) [inline Running-Mastodon/Production-guide.md]: It's unclear if this line is just for CentOS or for both ubuntu and CentOS

File: Running-Mastodon/Production-guide.md (hunk 1 above)
Comment (linuxfreeordie) [inline Running-Mastodon/Production-guide.md]: It wasn't removed it just got moved down. It should be more clear now with the newest commit.

File: Running-Mastodon/Production-guide.md (hunk 2 above)
Comment (d6rkaiz) [inline Running-Mastodon/Production-guide.md]: At least on debian does not have postgresql-setup.

**Comment by dariusk** [comment]
Reviewing these changes, I think the recent merge of #207 pretty much covers everything in here.
